logger.go

```diff
@@ -0,0 +1,9 @@
+// +build !gosnmp_debug
```

Comment:
This needs to be flipped around. This is a relatively small optimization, and most users will expect this to work by default.

```suggestion
// +build nogosnmp_debug
```

Reply:
Replaced with gosnmp_nodebug